front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -518,11 +521,41 @@ export function MessageView({
           </div>
           <div
             className={classNames(
-              "break-word ml-2  flex flex-1 flex-col whitespace-pre-wrap pt-1",
+              "break-word relative  ml-2 flex flex-1 flex-col whitespace-pre-wrap pt-1",
               message.role === "user" ? "italic text-gray-500" : "text-gray-700"
             )}
           >
             {toMarkdown(message)}
+            {message.role === "assistant" ? (
```

Comment:
NIT: should we have `message.role === "assistant" && message.message` instead? 
That would mean we don't need a fallback on `navigator.clipboard.writeText(message.message || "");` which makes sense as we don't want to offer a button to copy an empty string on the keyboard 😄

Reply:
Also NIT, but you can do {condition && ()} instead of {condition ? () : ""} 

Reply:
I avoid that form because if the condition fails the failing value is returned which sometimes leads to wrong output (e.g. 0 is a falsy value); most of the times navigators handle failing values well, e.g. they don't display undefined, but they don't have to => explicit behavior is IMHO better